tests/dialects/test_riscv.py

```diff
@@ -3,11 +3,16 @@
 
 
 def test_add_op():
-    a1 = TestSSAValue(riscv.RegisterType(riscv.Register()))
-    a2 = TestSSAValue(riscv.RegisterType(riscv.Register()))
-    add_op = riscv.AddOp(a1, a2)
+    a1 = TestSSAValue(riscv.RegisterType(riscv.Register.from_name("a1")))
```

Comment:
Would it make sense to support `Register("a1")` instead of using `from_name` to construct named registers?

Reply:
Thanks for fixing this.